eslzArm/managementGroupTemplates/policyAssignments/ENFORCE-LZDecommissionedPolicyAssignment.json

```
@@ -0,0 +1,59 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-08-01/managementGroupDeploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "topLevelManagementGroupPrefix": {
+            "type": "string",
+            "metadata": {
+                "description": "Provide the ESLZ company prefix to the intermediate root management group containing the policy definitions."
+            }
+        },
+        "enforcementMode": {
+            "type": "string",
+            "allowedValues": [
+                "Default",
+                "DoNotEnforce"
+            ],
+            "defaultValue": "Default"
+        },
+        "listOfResourceTypesAllowed": {
+            "type": "Array",
+            "defaultValue": [
+                "microsoft.consumption/tags"
+            ]
+        }
+    },
+    "variables": {
+        "policyDefinitions": {
+            "enforceLZDecommissioned": "[concat('/providers/Microsoft.Management/managementGroups/', parameters('topLevelManagementGroupPrefix'), '/providers/Microsoft.Authorization/policySetDefinitions/Enforce-LZ-Decomm')]"
+        },
+        "policyAssignmentNames": {
+            "lZDecommission": "Enforce-LZ-Decomm",
+            "description": "Enforce policies in the Decommisioned Landing Zone initiative.",
+            "displayName": "Enforce policies in the Decommisioned Landing Zone"
```

```suggestion
            "displayName": "Enforce ALZ Decommissioned Guardrails"
```
@matt-FFFFFF, @krowlandson, @jfaurskov, @JamJarchitect, @DaFitRobsta thoughts on this new policies for Decomissioned?